### ISSUE TITLE
Rewards page validations (wallets connected)

### ISSUE THREAD

**Issue by radoslaw-szpiech** [vegaprotocol/frontend-monorepo#765]
Linked to #477

**Comment by jtsang586** [comment]
There's been some roadblocks encountered implementing e2e tests for rewards page.

Attempted to use the vegwallet send command to top up the rewards pool but unable to see funds getting topped up at end of epoch.
Another roadblock is the need to enter wallet passphrase when submitting the send request which is cannot be done in command line using Cypress

Successful command:
`vega wallet command send --wallet capsule_wallet --pubkey 02eceaba4df2bef76ea10caf728d8a099a2aa846cced25737cccaa9812342f65 --network DV '{"transfer":{"fromAccountType":4,"toAccountType":12,"to":"0000000000000000000000000000000000000000000000000000000000000000","asset":"993ed98f4f770d91a796faab1738551193ba45c62341d20597df70fea6704ede","amount":"100000","recurring":{"startEpoch":1, "endEpoch": 100000000, "factor":"1"}}}' --home ~/.vegacapsule/testnet/wallet
Enter passphrase:

➜ Logs
2022-11-23T12:13:58.554Z	INFO	command	commands/command_send.go:297	retrieving block height
2022-11-23T12:13:58.554Z	INFO	forwarder	node/forwarder.go:199	Sending transaction to Vega node	{"host": "127.0.0.1:3002"}
2022-11-23T12:13:58.555Z	INFO	forwarder	node/forwarder.go:94		{"block-height": 4521, "block-hash": "b04f8f317381dd3b15dcfc66d7f947d91353648ed6cef73fa2d1a249190b2cc3", "difficulty": 15, "function": "sha3_24_rounds"}
2022-11-23T12:13:58.555Z	INFO	command	commands/command_send.go:307	last block height found: 4521
2022-11-23T12:13:58.555Z	INFO	command	commands/command_send.go:315	transaction successfully signed	{"signature": "4ee072ea20edc92fa638d8ae9eb919a988997c5841402aa964189f90f86b4f52e4eb7361c407e14fcc2342b85dfa034cf2b701dfe5231869989fbc6e322b450f"}
2022-11-23T12:13:58.564Z	INFO	command	commands/command_send.go:330	calculated proof of work for transaction	{"signature": "4ee072ea20edc92fa638d8ae9eb919a988997c5841402aa964189f90f86b4f52e4eb7361c407e14fcc2342b85dfa034cf2b701dfe5231869989fbc6e322b450f"}
2022-11-23T12:13:58.564Z	INFO	command	commands/command_send.go:343	transaction successfully sent	{"hash": "47438959EBA4677714123418F6918CED5073EDBD23AE4098D3D05B9E16C5C3F9"}
2022-11-23T12:13:58.564Z	INFO	forwarder	node/forwarder.go:60	gRPC clients successfully closed`

**Comment by pennyandrews** [comment]
Need to get some help from others on this?

**Comment by jtsang586** [comment]
Update on the current progress,
Managed to get rewards working on capsule which required going into both validator wallets and manually self delegating tokens in the UI.

I've attempted to use the wallet client to try and do this for E2E tests but I've run into a blocker where the wallet client still requires the wallet service to be running for each of those wallets and we currently have no way of stopping and restarting the wallet service in Cypress. 

I think it would be good to have a discussion to see what ways we can get around this problem.

Work that's been done is in the branch`test/rewards-e2e`
